### PR TITLE
SC-1671 Onboarding user v3 create API

### PR DESCRIPTION
**Pull request by anmol2302**
**Ticket Ref:** [SC-1671](https://project-sunbird.atlassian.net/browse/SC-1671?filter=10743)
**R2.8.0**
1: `/v3/user/create`(sync ES response) need to be onboarded.

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5127,3 +5127,20 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: createUserVersion3
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml]: can the 2 API's createUser and createUserVersion2 be deleted?

Comment (anmol2302) [reply ansible/roles/kong-api/defaults/main.yml]: @SMY-ALTAMASH no they are needed

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5127,3 +5127,20 @@ kong_apis:
...
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: "{{ createUserVersion3_ACL | default(['userUpdate']) }}"
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml]: do we require medium rate limiting for this API?

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml]: what is the reason for this API? 
Provide a description for this API.

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (anmol2302) [reply ansible/roles/kong-api/defaults/main.yml]: @SMY-ALTAMASH API does work same as user/v2/create.

**Pull request merged**